src/apis/users/services/users.service.ts

```diff
@@ -148,4 +150,70 @@ export class UsersService {
 
     return existUser;
   }
+
+  async putUpdate(
+    user: UserDto,
+    userId: number,
+    putUpdateUserDto: PutUpdateUserDto,
+  ) {
+    const myId = user.id;
+
+    if (!(myId === userId)) {
```

Comment:
```suggestion
    if (myId !== userId) {
```
보통 이렇게 많이 처리하는데 저렇게 짠 이유가 있나요?

Reply:
아.. 그러네요 원래 저렇게 안짜는데 너무 복잡하게 생각하면서 짜서 저렇게 된 것 같습니다.